client/verta/verta/_registry/modelversion.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+
+from .._tracking.entity import _ModelDBEntity
+
+
+class RegisteredModelVersion(_ModelDBEntity):
+    def __init__(self, conn, conf, msg):
+        raise NotImplementedError
+
+    def __repr__(self):
+        raise NotImplementedError
+
+    @property
+    def name(self):
+        self._refresh_cache()
+        return self._msg.name
+
+    @classmethod
+    def _generate_default_name(cls):
+        return "ModelVersion {}".format(_utils.generate_default_name())
+
+    @classmethod
+    def _get_proto_by_id(cls, conn, id):
+        raise NotImplementedError
+
+    @classmethod
+    def _get_proto_by_name(cls, conn, name, workspace):
+        raise NotImplementedError
+
+    @classmethod
+    def _create_proto_internal(cls, conn, ctx, name, desc=None, tags=None, attrs=None, date_created=None, public_within_org=None):
```

Comment:
@conradoverta My understanding is that `RegisteredModelVersion` being created from `RegisteredModel`s, rather than the `Client`, right?

Based on that, I don't think there's a way for it to receive a `ctx` arg, since that's limited to `Client`.

Reply:
is the ctx necessary?